### DISCUSSION TITLE
Static volume resize

### DISCUSSION THREAD

**Discussion by takyon77** [ceph/ceph-csi#3337] [Q&A]
Hi,

After an OKD cluster reinstallation, some cephfs subvolumes from an external Cephfs cluster dynamically created through ceph-csi were converted to static volumes to preserve users data. The problem is that it appears that volume resizing is not working on static volumes while it's working perfectly fine on dynamic volumes. I can do it manually and the method is quite easy so I'm surprised this is not possible automatically and I'm wondering if I did something wrong in the configuration (on the PVs side or in ceph-csi itself).
Is there a way to make resizing through ceph-csi possible on static volumes using subvolumes in Cephfs or is there a technical reason why ceph-csi is not able to do this?

Thanks.

**Comment by takyon77** [comment]
Ok I guess the problem here is that ceph-csi do not know that the static volume in this case is a cephfs subvolume and so it has no way to know how to handle a resize. Is there a way to configure the PVs or ceph-csi so it knows how to handle static volume resizing in this specific case?

**Reply by Madhu-1** [reply]
Yes, we dont support static volume operations except to mount and umount both for cephfs and rbd.
